docker-compose-examples/keycloak-postgres.yml

```diff
@@ -22,6 +22,7 @@ services:
         POSTGRES_PASSWORD: password
         KEYCLOAK_USER: admin
         KEYCLOAK_PASSWORD: Pa55w0rd
+        JDBC_PARAMS: "ssl=true"
```

Comment:
Not sure we should have ssl on by default. Rather just cover it in the readme.

Reply:
Same from the MySQL example, I thought it would be useful include a parameter in the example. Should we remove it?